icu4c/source/data/icu4j-readme.txt

```diff
@@ -17,8 +17,14 @@ In the following,
         $icu4j_root is the ICU4J root directory
         $jdk_bin is the JDK bin directory (for the jar tool)
 
-1. Download and build ICU4C. For more instructions on downloading and building
-        ICU4C, see the ICU4C readme at:
+1. Download, configure, and build ICU4C. When you configure ICU4C, you must
+   set the environment variable ICU_DATA_BUILDTOOL_OPTS to
+   "--include_uni_core_data" to build additional required ICU4J data:
+
+        ICU_DATA_BUILDTOOL_OPTS=--include_uni_core_data ./runConfigureICU Linux
```

Comment:
Forgive my ignorance, but does this also work for platforms without the runConfigureICU script?

Reply:
I put code to pass the same `ICU_DATA_BUILDTOOL_OPTS` environment variable as command-line arguments to Python in both configure.ac and makedata.mak, but I did not test it on any platforms other than Debian Linux.

Reply:
Thanks Shane. I don't have time to test this at the moment, but it feels like we can always just file a bug in the future if need be. thanks.